operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -1,263 +1,371 @@
+
+
 ## Add LDAP User Federation
 
 Add LDAP user federation using the Keycloak localization tool.
 
+
 ### Prerequisites
 
 LDAP user federation is not currently configured in Keycloak. For example, if it was not configured in Keycloak when the system was initially installed or the LDAP user federation was removed.
 
-### Procedure
-
-1.  Update the LDAP settings in the customizations.yaml file.
-
-    The LDAP server CA certificate goes into SealedSecret. Refer to the "Generate Sealed Secrets" header in the [Prepare Site Init](../../install/prepare_site_init.md) procedure for instructions on how to do that.
```

Comment:
This was removed but it still something that the customer might need to do. If they need to load the LDAP server CA certificate (e.g., if they're using ldaps and the server's host cert isn't already trusted by the built-in host certs).

Reply:
Steps 4-5 below were pulled in from the "Generate Sealed Secrets" header in the prepare_site_init.md file.